src/vk_mem_alloc.h

```diff
@@ -15106,9 +15105,8 @@ VmaRecorder::UserDataString::UserDataString(VmaAllocationCreateFlags allocFlags,
         else
         {
             // If VMA_ALLOCATION_CREATE_USER_DATA_COPY_STRING_BIT is not specified, convert the string's memory address to a std::string and store it.
-            std::stringstream user_data_address_to_string_converter;
-            user_data_address_to_string_converter << pUserData;
-            m_Str = user_data_address_to_string_converter.str();
+			snprintf(m_PtrStr, 17, "%p", pUserData);
```

Comment:
This is C++11's standard version of sprintf_s.

Reply:
You should use `sizeof(m_PtrStr)` instead of 17 so it automatically stays in sync if someone changes it

Reply:
Let's not over-generalize. I don't expect the size of a pointer to go beyond 64 bits in the following years, at least on typical PC and mobile platforms.